src/docs/patterns/tables.mdx

```diff
@@ -0,0 +1,463 @@
+        '2: In a list, users rely heavily on a primary property to navigate (layer cake pattern).',
+    ]}
+/>
+
+If you expect your users to compare across multiple parameters before they can decide which row to interact with, you should use a table. If they mostly read your data row-by-row, you shall try structuring your data as a list.
+
+<ImageGrid
+    images={[ColumnOrder1, ColumnOrder2]}
+    caption={[
+        `Users are more likely to find a user by their name or their role, not by their email address or phone number. Therefore, you should arrange the name and the role columns to the left, followed by phone number and email (#1). Email addresses and phone numbers should not show up before the role column (#2).`,
+    ]}
+    captionsUnderImages={['', 'DONT:']}
+/>
+
+## Property Name
+
+In tables, you would call out properties of each data entry only once in the header row. In lists, however, either the values are clear enough on their own without explanations, or property names have to be included in every list item.
+
+<ImageGrid
+    images={[ImplicitValueTable]}
+    captionsUnderImages={[`The values are explained by the table header row.`]}
+    gridContainerProps={{ style: { justifyContent: 'flex-start' } }}
+/>
+<ImageGrid
+    images={[ImplicitValueList1, ImplicitValueList2]}
+    captionsUnderImages={[
+        `DONT: It is not clear what those values in the subtitle mean.`,
+        `CAUTION: Be careful when calling out property names in lists, as they weaken the scanability.`,
+    ]}
+/>
+
+## Consumer-Facing Applications
+
+Avoid using tables in consumer-facing applications. Lists mentally match real-world individual objects, while tables can be more abstract. Use tables only if your users really need to examine technical details.
+
+<Divider />
+
+# Responsive
+
+In general, **we do not encourage tables on mobile platforms**, as table-related interactions tend to be complicated for small viewports, and mobile phones also often lack the computational power to process massive data set.
+
+<ImageGrid
+    images={[TableOnMobile]}
+    regularWidth
+    caption={`Users can only see a very small portion of the table through mobile devices.`}
+/>
+
+## Understanding Users' Goal
+
+<ImageGrid images={[UserInField]} regularWidth caption={`While on foot, it is very hard to examine small details.`} />
+
+Before you design a table structure for a mobile application, you should conduct user researches and have a clear understanding of users' intentions. When they land on this page, Do they desire to do complicated cross-comparison between multiple rows and columns? Is a full table necessary?
+
+For industrial applications, your users would often be using a mobile app when they are out in the field. They are paying more attention to their surroundings, so keep the mobile app interaction minimal.
```

Comment:
This is not technically correct. There are a lot of industrial/monitoring applications that are not mobile. I would content that mobile apps are the new kid on the block in that regard.

Reply:
Meant to say that, for industrial mobile apps, when your users are using them, they are often on foot. 🙈